worlds/archipidle/Rules.py

```diff
@@ -14,26 +14,23 @@ def _archipidle_location_is_accessible(self, player_id, items_required):
 
 
 def set_rules(world: MultiWorld, player: int):
-    for i in range(1, 16):
-        set_rule(
-            world.get_location(f"IDLE for at least {int(i / 2)} minutes {30 if (i % 2) > 0 else 0} seconds", player),
-            lambda state: state._archipidle_location_is_accessible(player, 0)
-        )
-
```

Comment:
why is this being removed? can you clarify?

Reply:
Because the rule there is essentially `lambda true` but it wastes a bunch of time going through all the locations.